howtos/run_conan_in_docker.rst

```diff
@@ -0,0 +1,181 @@
+
+    docker run -it --rm lasote/conangcc7 /bin/bash
+
+The previous code will run a shell in container. We have specified:
+
+- ``-it``: Keep STDIN open and allocate a pseudo-tty, in other words, we want to type in the container because we are opening a bash.
+- ``--rm``: Once the container exits, remove the container. Helps to keep clean or hard drive.
+- ``lasote/conangcc7``: Image name, check the :ref:`available docker images<available_docker_images>`.
+- ``/bin/bash``: The command to run
+
+
+Now we are running on the conangcc7 container we can use Conan normally. It is always recommended
+to upgrade conan from pip first:
+
+.. code-block:: bash
+
+    $ sudo pip install conan --upgrade
+    $ git clone https://github.com/conan-community/conan-openssl
+    $ cd conan-openssl
+    $ conan create . user/channel
+
+
+Sharing a local folder with a docker container
+----------------------------------------------
+
+You can share a local folder with your container, for example a project:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-community/conan-openssl
+    $ cd conan-openssl
+    $ docker run -it -v$(pwd):/home/conan/project --rm lasote/conangcc7 /bin/bash
+
+
+- ``v$(pwd):/home/conan/project``: We are mapping the current directory (conan-openssl) to the container
+  ``/home/conan/project`` directory, so anything we change in this shared folder, will be really changed
+  in our host machine.
+
+.. code-block:: bash
+
+    # Now we are running on the conangcc7 container
+    $ sudo pip install conan --upgrade
+    $ cd project
+    $ conan create . user/channel --build missing
+    $ conan remote add myremote http://some.remote.url
+    $ conan upload "*" -r myremote --all
+
+
+Using the images to cross-build packages
+----------------------------------------
+
+You can use the :ref:`images<available_docker_images>` ``-i386``, ``-armv7`` and ``-armv7gh`` to cross build
+conan packages.
+
+The ``armv7`` images have a cross toolchain for linux ARM installed, and declared as main compiler with the
+environment variables ``CC`` and ``CXX``. Also, the default Conan profile (``~/.conan/profiles/default``)
+is adjusted to declare the correct arch (``armv7`` / ``armv7hf``).
+
+
+.. code-block:: bash
+
+    $ cat ~/.conan/profiles/default
+
+    [settings]
+    os=Linux
+    os_build=Linux
+    arch=armv7hf
+    arch_build=x86_64
+    compiler=gcc
+    compiler.version=4.9
+    compiler.libcxx=libstdc++
+    build_type=Release
+    [options]
+    [build_requires]
+    [env]
+
+We can cross build a package and all the missing dependencies for ``Linux/armv7hf`` and
+finally uploading the built packages to a repository:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-community/conan-openssl
+    $ cd conan-openssl
+    $ docker run -it -v$(pwd):/home/conan/project --rm lasote/conangcc49-armv7hf /bin/bash
+
+    # Now we are running on the conangcc49-armv7hf container
+    $ sudo pip install conan --upgrade
```

Comment:
Add a comment like:

```
$ sudo pip install conan --upgrade # We make sure we are running the latest Conan version
```

Or something like that. I supposse conan is already installed as you said above.

Reply:
Yes, now I see you said it above. I leave this change up to you 😆